### ISSUE TITLE
Refresh stops when items are dropped from the cart

### ISSUE THREAD

**Issue by oby1** [lastquestion/reload-amazon-fresh-chrome-ext#3]

**Comment by lastquestion** [comment]
Hm. Why do items get dropped from the cart? Is it when they go out of stock? I don't know how to deal with it, does the URL change?

**Comment by oby1** [comment]
Yes, they go out of stock rather frequently now due to the high demand.  I believe the URL changes but I don't have the details.  I'll update the ticket with more information once I'm able to reproduce the issue.
